cogment_verse/web/web_app/src/components/RenderedScreen.module.css

```diff
@@ -17,4 +19,4 @@
   display: flex;
   justify-content: center;
   align-items: center;
-}
+}
```

Comment:
@lhnguyen102 please setup your editor to use the .editorconfig at the root, trailing lines should be the standard.

Reply:
Done